src/components/Banners/Banner/BannerLabels/Labels.tsx

```diff
@@ -0,0 +1,193 @@
+//- Component Imports
+import { Countdown } from 'components';
+
+//- Container Imports
+import { Stage } from 'containers/flows/MintWheels/types';
+
+//- Utils Imports
+import { IndustryType, getBannerTitle } from '../utils';
+
+const totalLabel = (itemsMinted: number, totalItems: number) => (
+	<span>
+		<b>{totalItems - itemsMinted} Remaining</b>
+	</span>
+);
+
+//- Raffle Event Message
+export const getRaffleBannerLabel = (
```

Comment:
I was thinking we could determine whether the ```BannerEventType``` is Raffle or Mint. My thought process here would be that we could rename ```WheelsRaffle.tsx``` to ```Raffle.tsx```, and ```MintWheels.tsx``` to ```Mint.tsx``` and make these generic and when used pass the Industry Type.

 From a higher level if we can toggle Industry Type it can be used to customise the text as structured below.